### PR TITLE
mojito: improve power profiles

### PR DESCRIPTION
**Pull request by ghost**
Values taken from RN11
> Set WiFi voltage
> Tweak modem & radio controllers
> Fix cpu suspend soft reboots after certain idle time limit
> Set gps signalqualitybased

Signed-off-by: Prince Cooper <83843264+pc00per@users.noreply.github.com>

### CODE REVIEW

File: Xiaomi/RedmiNote10/res/xml/power_profile.xml [hunk 1]
```diff
@@ -57,54 +57,48 @@
...
-        <value>.2</value>
-        <value>2</value>
+        <value>.0001</value>
+        <value>.001</value>
+        <value>.01</value>
+        <value>.1</value>
+        <value>1</value>
     </array>
-    <array name="wifi.controller.tx_levels" />
+    <array name="wifi.controller.tx_levels">1</array>
```
Comment (phhusson) [inline Xiaomi/RedmiNote10/res/xml/power_profile.xml]: Array with 0 item?

**Comment by phhusson** [comment]
Values are from where?
>  Fix cpu suspend soft reboots after certain idle time limit
How?

> Improve WiFi range & set voltage

You can't improve wifi range through power_profile

File: Xiaomi/RedmiNote10/res/xml/power_profile.xml (hunk 1 above)
Comment (ghost) [reply Xiaomi/RedmiNote10/res/xml/power_profile.xml]: Oh that wasn't needed at all btw.

**Comment by ghost** [comment]
> Values are from where?

Stock power_profiles from framework_res are encrypted. Device tree values give soft reboot issues, when device left idle for hours.
These values extracted from RN11, which has the same hardware.
> You can't improve wifi range through power_profile

Yah lmao, that was just a buff. Values taken from rn11 & tested.

**Comment by pc00per** [comment]
@phhusson

**Pull request closed without merging**